app/src/main/res/layout/main_fragment.xml

```diff
@@ -82,4 +71,15 @@
             android:gravity="center_horizontal"
              />
 
+        <ImageButton
+            android:id="@+id/btnShoppingList"
+            android:layout_width="100dp"
+            android:layout_height="wrap_content"
+            android:layout_marginStart="16dp"
+            android:layout_marginLeft="16dp"
+            android:src="@drawable/ic_shopping_list"
+            app:layout_constraintStart_toStartOf="parent"
+            tools:layout_editor_absoluteY="105dp" />
+
+
 </androidx.constraintlayout.widget.ConstraintLayout>
```

Comment:
I added a image button, on my screen the text can be a little hard to read due how small it was  ( may have been my monitor) but I added a image button with a shopping cart to take the place of the button with the text of "Shopping List".  This changes change help with people who may not know the language that well  but can understand images any where

Reply:
Pictures are faster than words.  Great idea.